interface-elements-for-web/articles/scheduler/appointment-management/make-a-series-of-recurring-appointments.md

```diff
@@ -5,38 +5,40 @@ legacyId: 8459
 ---
 # Make a Series of Recurring Appointments
 ## What is a Recurring Appointment?
-All appointments supported by the **Scheduler** control can be simple (non-recurring) or recurring. Non-recurring appointments occur only once in a specified time interval. Recurring appointments occur many times in the same time interval on different dates.
+All appointments the **Scheduler** control supports can be simple (non-recurring) or recurring. Non-recurring appointments occur only once at a specified time interval. Recurring appointments occur many times at the same time interval on different dates.
 
-Recurring appointments are created as series of appointments, specified by the [recurrence pattern](#recurrencepattern) and indicated with the ![AppointmentImageType.Recurrence](../../../images/img4572.png) recurrence sign.
+Recurring appointments are created as a series of appointments using the [recurrence pattern](#recurrencepattern) and identified by the ![AppointmentImageType.Recurrence](../../../images/img4572.png) recurrence sign.
 
 ![RecurringAppointments](../../../images/img12067.png)
 
 ## <a name="recurrencepattern"/>Recurrence Pattern
 The recurrence pattern contains information on appointment recurrence (a start date and time of the recurrence series and a rule according to which a series is repeated).
```

Comment:
>a start date and time of the recurrence series and a rule according to which a series is repeated" What are you trying to say here?

This is information about each recurring appointment in the recurrence series. A start date and time - when an appointment occurs.